lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2104,8 +2070,7 @@ def contour(self, X, Y, Z, *args,
         Parameters
         ----------
         X, Y, Z : array-like,
-            Input data. See `~matplotlib.axes.Axes.contour` for acceptable
-            data shapes.
+            Input data. See `.Axes.contour` for acceptable data shapes.
```

Comment:
Is "accepted" the correct word here? Or should it be "valid" or "supported"?

If not applies to the other *`contour*` docstrings as well.

Reply:
Switched to supported, which does read better.